spec/controllers/users/piv_cac_authentication_setup_controller_spec.rb

```diff
@@ -111,7 +111,7 @@
         context 'when redirected with an error token' do
           it 'renders the error template' do
             get :new, params: { token: bad_token }
-            expect(response).to render_template(:error)
+            expect(response).to redirect_to setup_piv_cac_path
```

Comment:
Can we move this to a feature spec, please? We learned from a recent production 500 error that controller specs that don't render views will not catch any issues with views that call objects such as presenters.